.cbmc-batch/jobs/aws_priority_queue_s_sift_up/Makefile

```diff
@@ -0,0 +1,50 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+# this file except in compliance with the License. A copy of the License is
+# located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+###########
+include ../Makefile.aws_array_list
+include ../Makefile.aws_priority_queue_sift
+
+###########
+#
+# Runtime:
+# - 300s for MAX_PRIORITY_QUEUE_ITEMS=3 items
+# - 450s for MAX_PRIORITY_QUEUE_ITEMS=5 items 
+
+
+# Note:
+# In order to reach full coverage we have to unwind
+# log(NUMBER_PRIO_QUEUE_ITEMS) but this number is huge so we only
```

Comment:
How can a `log` of a machine integer ever be "huge?"

Reply:
You are right, that is a wrong and outdated comment that slipped through. I will correct it.